docs/Contributing/architecture/orchestration/orchestration-overview.md

```diff
@@ -0,0 +1,65 @@
+# Orchestration Architecture Overview
+
+This document provides an overview of Fleet's Orchestration architecture.
+
+## Introduction
+
+Fleet's Orchestration architecture is designed to manage and query devices at scale using osquery, providing visibility into device status, configuration, and security posture. This document provides insights into the design decisions, system components, and interactions specific to the Orchestration functionality.
+
+## System Components
+
+The Orchestration architecture consists of the following main components:
+
+- **Fleet Server**: The central component that manages device communication, query execution, and result processing.
+- **osquery**: The open-source agent that runs on devices and executes queries.
+- **Query Engine**: The component that processes and executes queries on devices.
+- **Result Processing**: The component that processes and stores query results.
+- **Teams and Access Control**: The component that manages user access to devices and queries.
```

Comment:
This doesn't feel accurate/isn't how we think of components of the system...and the "query engine" is osquery, and orbit isn't mentioned at all here.

Reply:
Agreeds needs review and editing